modules/ossm-config-otel.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/traces/ossm-distr-tracing-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-otel_{context}"]
+= Configuring {OTELName} with Service Mesh
+
+You can integrate {SMProductName} with {OTELName} to instrument, generate, collect, and export telemetry traces, metrics, and logs to analyze and understand your software's performance and behavior.
+
+.Prerequisites
+
+* A TempoStack is installed and configured as described here: link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/distributed_tracing/distributed-tracing-platform-tempo#installing-a-tempostack-instance[Installing a TempoStack instance].
+* An instance of an `OpenTelemetry` collector is already configured in the `istio-system` namespace.
+* An Istio instance is created in the `Istio` resource (`Istio.spec.profile`).
+* An Istio CNI instance is created in the `Istio` resource (`Istio.spec.profile`).
+
+.Procedure
+
+. Configure {SMProductName} to enable tracing, and define the {OTELShortName} tracing providers in your `meshConfig`:
++
+.Example `meshConfig` for tracing
+[source,yaml]
+----
+apiVersion: sailoperator.io/v1alpha1
+kind: Istio
+metadata:
+  ...
+  name: default
+spec:
+  namespace: istio-system
+  ...
+  values:
```

Comment:
Question... why when I look at the preview, does it only show: 
```
meshConfig:
  enableTracing: true
  extensionProviders:
  - name: otel-tracing
    opentelemetry:
      port: 4317
      service: otel-collector.istio-system.svc.cluster.local
 ```
But here it shows most of the Istio config? Is that a bug? As a user, I'd be a bit lost with just the meshConfig block - where does that go? You could also mention it in the preceding sentence... "tracing providers in your `Istio`'s `meshConfig`:".

Reply:
Oh weird, I reloaded and it appeared properly - never mind. :)

Reply:
Likely you look at the preview before the build finished. Once the build finishes (assuming no failures), the preview is updated accordingly.